frontend/src/components/pages/details/components/DetailsGear/DetailsGear.tsx

```diff
@@ -19,7 +18,7 @@ export const DetailsGear: React.FC<DetailsGearProps> = ({ className, text }) =>
         <BackPack />
       </div>
       <div className="text-greyDarkColored desktop:font-bold text-Mobile-C2 desktop:text-P1 my-auto">
-        <HtmlText>{parse(text)}</HtmlText>
+        <div className="content-WYSIWYG">{parse(text)}</div>
```

Comment:
As `HtmlText` is used multiple times, i would have leave it, but simply re-authored like:

```js
const HtmlText = ({ className = '', ...props }) => (
  <div
    className={cn(className, 'content-WYSIWYG')}
    {...props}
  />
);
```